product-service-description/compliance-as-a-service/products/middleware.md

```diff
@@ -9,10 +9,7 @@ Die lokal installierte fiskaltrust.Middleware bietet die Einbindung von TSE alle
 
 ![middleware](../media/middleware.png)
 
-Die fiskaltrust.Middleware besteht aus 
-
-- einer Launcher App zum Installieren der Middleware lokal auf dem Kassenrechner
-- einem Middleware Dienstprogramm (Service/Daemon; läuft dauerhaft auf dem Kassenrechner)
+Die fiskaltrust.Middleware besteht aus einem Launcher welcher die Middleware startet und dauerhaft als Service/Daemon auf dem Rechner bzw. dem Kassensystem betreibt, sowie auch für etwaige Aktualisierungen der Version sorgt.
```

Comment:
```suggestion
Die fiskaltrust.Middleware besteht aus einem fiskaltrust.Launcher welcher die Bestandteile beinhaltet, die Middleware zu starten und länderabhängig dauerhaft bzw. als Windows-Service/Unix-Daemon/Android-Service auf dem POS-System, in einem Rechenzentrum oder in der fiskaltrust.Cloud betrieben wird, sowie auch für etwaige Aktualisierungen der Version sorgt. Die Aktualisierung des lokalen fiskaltrust.Launchers selbst ist nur durch neuerlichen Download aus dem fiskaltrust.Portal und manuellen Tausch der Installation möglich. 
```

Reply:
1. Guter Punkt.
2. Ein Service ist bei Android etwas anderes und gibt es in dem Sinn nicht.
3. Die ganze Produktbeschreibung sollte vermutlich eines Tages auch auf Englisch verfügbar sein. Momentan ist sie nur für den deutschen Markt auf Deutsch verfügbar.

Textvorschläge noch eingebaut, mit leichten Änderungen/Versimplifizierungen.

Reply:
@2. Ist es vielleicht ein "Android-Prozess"? Das könnte auch ergänzt werden.

Reply:
Ist Teil der Android Applikation. Ich würd hier aber gar nicht so sehr ins Detail gehen. Respektive lädt er unter Android ja auch nichts runter. Würd das noch anpassen.